app/components/provider_interface/personal_details_component.rb

```diff
@@ -32,6 +32,7 @@ def rows
         phone_number_row,
         email_row,
         address_row,
+        candidate_id_row,
```

Comment:
not sure if we want to mirror this in the support application details component, although support does surface the candidate id elsewhere on that page 🤔

Reply:
I think it's ok as it is — right up at the top of the page in Support